mock/src/mock/transaction.rs

```diff
@@ -53,17 +51,13 @@ pub fn mock_inputs_with_account_seed(
         },
     };
 
-    // mock notes
```

Comment:
These comments didn't add context, the function being called is already named `mock_notes`.